app/src/main/java/com/pajato/android/gamechat/database/AccountManager.java

```diff
@@ -336,8 +349,19 @@ private void register() {
 
     /** Sign in using the saved User account. */
     private void signIn(final Context context) {
-        // Invoke the sign in activity to kick off a Firebase auth event.
-        Intent intent = new Intent(context, SignInActivity.class);
+        // Get an instance of AuthUI based on the default app, and build an intent.
```

Comment:
Actually, it might be even better to have `getSignInIntent()` which returns the built intent.